lib/materials/data_helpers.rb

```diff
@@ -111,6 +137,7 @@ def materials_data(materials, assigned_to_class = nil)
 
           subject_areas:    tags['subject_areas'],
           grade_levels:     tags['grade_levels'],
+          projects:         projects,
```

Comment:
Instead of a simple project name. The projects should be an array of objects. Each object should have the following fields:
- name
- id
- landing_page_url : this is only set if there is a landing_page_slug and it includes the slug. For example: http://github.com/concord-consortium/rigse/blob/d3b960230784f3b4f19650bd28e9eb62b89ef92f/app/views/admin/projects/_show.html.haml#L20
- public : true or false value